.github/workflows/checks-codecov.yaml

```
@@ -34,6 +34,7 @@ permissions:
 jobs:
 
   Test:
+    if: false  # skipped for POC branch
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)